packages/dev/core/src/ShadersWGSL/ShadersInclude/pbrDebug.fx

```diff
@@ -22,9 +22,9 @@ if (input.vClipSpacePosition.x / input.vClipSpacePosition.w >= uniforms.vDebugMo
         color = normalW;
         #define DEBUGMODE_NORMALIZE
     #elif DEBUGMODE == 6 && defined(MAINUV1)
-        color =  vec3f(vMainUV1, 0.0);
+        color =  vec3f(input.vMainUV1, 0.0);

```

Comment:
`fragmentInputs` ?

Reply:
both are defined and the same (Something I will need to clean-up later if this becomes a problem but it was there already)

Reply:
I did not change it to be fair because NME depends on both :(